events/courtier_guest_management_events/courtier_guest_management_events.txt

```diff
@@ -828,6 +828,10 @@ scripted_trigger courtier_guest_management_3001_interesting_trigger = {
 		has_trait = intellect_good
```

Comment:
Guests get interesting traits here and magic traits aren't included.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105056168-56517680-5a8d-11eb-91f9-464f4d387f4d.png)

</details>